modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -308,19 +308,31 @@ When using the `neo4j-admin database backup` command, you can configure the back
 For more information on how to configure SSL in Neo4j, see xref:security/ssl-framework.adoc[SSL framework].
 
 Configuration for the backup server should be added to the _neo4j.conf_ file and configuration for backup client to the _neo4j-admin.conf_ file.
-SSL settings should be set identically between both to ensure compatibility.
+The easiest way to ensure compatibility is to use the same SSL policy configuration for both the server and the client.
+However, this approach is not recommended for production environments, where it is better to use Certificate Authorities (CAs) to sign the certificates used by both the server and the client.
+For details, see xref:security/ssl-framework.adoc[SSL framework].
 
 The default backup port is 6362, configured with key `server.backup.listen_address`.
 The SSL configuration policy has the key of `dbms.ssl.policy.backup`.
 
 As an example, add the following content to your _neo4j.conf_ and _neo4j-admin.conf_ files:
 
+.Server configuration in _neo4j.conf_
 [source, properties]
 ----
 dbms.ssl.policy.backup.enabled=true
-dbms.ssl.policy.backup.tls_versions=TLSv1.2
-dbms.ssl.policy.backup.ciphers=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
 dbms.ssl.policy.backup.client_auth=REQUIRE
+dbms.ssl.policy.backup.tls_versions=TLSv1.2,TLSv1.3
+dbms.ssl.policy.backup.ciphers=TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
```

Comment:
I would exclude the last item on the cipher list in the example as it is considered a weak cipher https://ciphersuite.info/cs/TLS_DHE_RSA_WITH_AES_256_GCM_SHA384/